opm/autodiff/StandardWell_impl.hpp

```diff
@@ -884,6 +881,18 @@ namespace Opm
             }
         }
 
+        // F_solvent is added to F_gas. This means that well_rate[Gas] also contains solvent.
+        // More testing is needed to make sure this is correct for well groups and THP.
+        if (has_solvent){
+            const double scal = scalingFactor(contiSolventEqIdx);
+            if (scal > 0) {
```

Comment:
Under what kind of circumstances, `scal` is zero here with `has_solvent` to be true?  Maybe I misunderstood the details of function `scalingFactor`.

Reply:
Never. I just copied this from above without thinking. I will remove it.